zenodo/base/templates/zenodo/faq.html

```diff
@@ -70,23 +66,23 @@
+        <p>Since there isn't a unique way of licensing openly and nor a consensus on the practice of adding attribution restrictions, we accept data under a variety of licenses in order to be inclusive. We will, however, take an active lead in signaling the extra benefits of the most open licenses, in terms of visibility and credit, and offer additional services and upload quotas on such data to encourage using them. This follows naturally from the publications policy of the <a href="http://www.openaire.eu">OpenAIRE initiative</a>, which has been supporting Open Access throughout, but since it aims to gather all European Commission/European Research Area research results, it allows submission of material that is not yet Open Access.</p>
     </li>
     <li>
         <h5>What happened to the OpenAIRE Orphan Record Repository?</h5>
-        <p>OpenAIRE Orphan Record Repository got a make-over and was re-branded as {{config.CFG_SITE_NAME}}. If you happened to deposit your article in OpenAIRE Orphan Record Repository it is also available in {{config.CFG_SITE_NAME}}. You user account, was however not transferred to {{config.CFG_SITE_NAME}}, so you will have to <a href="{{url_for('youraccount.register')}}">register</a> again. If you register with the same email address in {{config.CFG_SITE_NAME}} as you used in OpenAIRE Orphan Record Repository, you will still have access to your publications. Don't hesitate to <a href="{{url_for('zenodo_base.contact')}}">contact us</a> for further information.</p>
+        <p>OpenAIRE Orphan Record Repository got a make-over and was re-branded as {{config.CFG_SITE_NAME}}. If you deposited your article in OpenAIRE Orphan Record Repository, it is also available in {{config.CFG_SITE_NAME}}. However, your user account was not transferred to {{config.CFG_SITE_NAME}}, so you will have to <a href="{{url_for('youraccount.register')}}">register</a> again. If you register with the same email address in {{config.CFG_SITE_NAME}} as you used in OpenAIRE Orphan Record Repository, you will still have access to your publications. Don't hesitate to <a href="{{url_for('zenodo_base.contact')}}">contact us </a>for further information.</p>
     </li>
     <li>
         <h5>Where does the name come from?</h5>
         <p>{{config.CFG_SITE_NAME}} is derived from <a href="http://en.wikipedia.org/wiki/Zenodotus">Zenodotus</a>, the first librarian of the Ancient Library of Alexandria and father of the first recorded use of metadata, a landmark in library history.</p>
     </li>
     <li>
-        <h5>How much storage do CERN have available?</h5>
-        <p>{{config.CFG_SITE_NAME}} is currently a drop in the ocean. CERN currently stores more than 100PB of physics data from the <a href="http://home.web.cern.ch/about/accelerators/large-hadron-collider">Large Hadron Collider (LHC)</a>, and produces roughly 25PB per year when the LHC is running.</p>
+        <h5>How much digital storage-space does CERN have available?</h5>
```

Comment:
Not sure I would go with: ``digital storage-space``. At least it is not a normal term in this context. Also, we cannot use e.g. ``disk space`` since the space is both on disk and tape.